draft-ietf-quic-recovery.md

```diff
@@ -392,6 +357,72 @@ Pseudocode for OnPacketAcked follows:
      sent_packets.remove(acked_packet);
 ~~~
 
+## Setting the Loss Detection Alarm
+
+QUIC loss detection uses a single alarm for all timer-based loss detection.  The
+duration of the alarm is based on the alarm's mode, which is set in the packet
+and timer events further below.  The function SetLossDetectionAlarm defined
+below shows how the single timer is set based on the alarm mode.
+
+### Handshake Packets
+
+The initial client hello has no prior RTT sample.  A client SHOULD remember
+the previous RTT it observed when resumption is attempted and use that for an
+initial RTT value.  If no previous RTT is available, the initial RTT defaults
+to 200ms.  Once an RTT measurement is taken, it MUST replace initial_rtt.
+
+The client hello is retransmitted via a timer, similar to a tail loss probe, but
```

Comment:
@martinthomson Yeah, maybe "For each previous handshake restransmission on this connection, the time limit doubles".

The way I wrote this actually implies that the timeout doubling doesn't actually reset, i.e. chlo times out twice, causing the timeout to double. Then the Finished is lost, and remains at the doubled length. I'm not sure if I meant to do that, but be aware of it.